Source/WebCore/animation/TimelineScope.h

```diff
@@ -34,7 +34,7 @@ struct TimelineScope {
     bool operator==(const TimelineScope& other) const
     {
         if (type == Type::Ident)
-            return other.type == Type::Ident && scopeNames != other.scopeNames;
+            return other.type == Type::Ident && scopeNames == other.scopeNames;
         return type == other.type;
     }
```

Comment:
Just using the default operator would probably have avoided this bug.

Reply:
Indeed, that works as well, I'll use the default implementation.